frontend/src/pushapi.spec.js

```diff
@@ -6,68 +6,73 @@ describe('PushApi', function () {
 
   beforeEach(module('clouway-push'));
 
-  var pushApi, socket, rootScope, connectMethod, bindMethod, unbindMethod;
+  var pushApi, socket, rootScope;
   var subscriber = 'test-subscriber';
   var channelToken = 'fake-channel-token';
   var $window = {};
 
-  describe('service should', function () {
+  describe('after established connection ', function () {
+     var $interval, $timeout, httpBackend;
+     var keepAliveInterval = 5; //in seconds
+     var reconnectInterval = 20; //in seconds
```

Comment:
Seems that in this describe block the `rootScope`, `$window`, `$interval`, `$timeout` and `reconnectInterval` are not used.